common/libs/transaction.js

```diff
@@ -121,13 +122,14 @@ export async function generateTransaction(
 
   // Generate the raw transaction
   const txCount = await node.getTransactionCount(tx.from);
+  const cleanHex = hex => addHexPrefix(padToEven(stripHex(hex)));
```

Comment:
What kind of padding are we doing here? Maybe some comments to explain?

Reply:
This was taken from v3's `sanitizeHex` behavior, Trezor wanted %2 data. I can definitely add some comments to v3, but I don't truly understand the reasoning.